src/components/SelectDevice/index.js

```diff
@@ -201,18 +214,23 @@ class SelectDevice extends Component<OwnProps, State> {
     );
 
     const hasUSBSection = Platform.OS === "android" || other.length > 0;
+    const isNanoSOnOnboarding = isNanoS && onboarding;
 
     return (
       <View>
-        {ble.length === 0 ? (
+        {isNanoS && onboarding ? (
```

Comment:
I think the condition can just be `isNanoS` , you can also remove `isNanoSOnOnboarding`